database.sql

```diff
@@ -103,12 +103,12 @@ CREATE TABLE `Course` (
 -- Structure de la table `Discipline`
 --
 
-CREATE TABLE 'Discipline' (
-  'id' INT NOT NULL,
-  'icon' VARCHAR(100) NOT NULL,
-  'name' VARCHAR(100) NOT NULL,
-  'description' VARCHAR(5000) NOT NULL,
-  PRIMARY KEY ('id')
+CREATE TABLE `Discipline` (
+  `id` INT NOT NULL,
```

Comment:
id should be auto increment

Reply:
it is  already auto increment
****
- AUTO_INCREMENT pour la table `Discipline`
--
ALTER TABLE `Discipline`
  MODIFY `id` INT NOT NULL AUTO_INCREMENT;

Reply:
I changed that, with deleting all the alter request + add auto increment in all tables